src/components/lib/dataParser.js

```diff
@@ -151,7 +151,7 @@ export function parseData(data, mapData, colors, fields, layer) {
         parsedData.push({
           in: row[srcKey],
           out: row[dstKey],
-          azName: `${row[srcKey]}---${row[dstKey]}`,
+          azName: `${row[srcKey]}--${row[dstKey]}`,
```

Comment:
this change (`---` to `--`) would seem to have far-reaching implications in this codebase.

Reply:
Triple hyphens restored.